CONTRIBUTING.md

```diff
@@ -87,3 +87,92 @@ This part of the development documentation should be a full reference of all ava
 #### Cookbook
 
 This should contain specific examples and use cases on how to use and implement the previously mentioned topics.
+
+
+## Some notes about your markdown contribution
+
+* The paths and filenames should be english
+* Consider that all markdown headings are taken over in the page navigation of the documentation
+* Use images by copieng existing or create new ones only
+
+
+## The hugo front matter variables
+
+Every file needs at least the following [hugo front matter](https://gohugo.io/content-management/front-matter/) variables on top of your markdown content:
+
+* `title`: A title for the file (language specific). This entry refletcs the main left navigation of the documentation also.
+* `description`: The content description (language specific).
+* `url`: The url path to build (language specific).
```

Comment:
This should only be used, if you want the path to be different from the physical path. e.g. for the German translation.